GoogleForCreators/web-stories-wp
# UI Simplifications: Border Panel

## Summary

Updates and simplifies the border width inputs design for locked and unlocked status





## To-do

Changes for the border color eyedropper  are addressed on a different PR

## User-facing changes



| Before | After |
| ------ | ----- |
| <img width="306" alt="Screenshot 2021-10-26 at 17 27 41" src="https://user-images.githubusercontent.com/276316/139306606-e15c2316-6c5c-47fd-b245-6ce82bc1612a.png"> | <img width="301" alt="Screenshot 2021-10-26 at 17 26 08" src="https://user-images.githubusercontent.com/276316/139307500-bd75e63f-c4f4-4a29-9ad5-917c4335a9c3.png"> |
| <img width="302" alt="Screenshot 2021-10-26 at 17 27 50" src="https://user-images.githubusercontent.com/276316/139306911-47b78ef1-404d-4844-bab5-028407c69ede.png"> | <img width="305" alt="Screenshot 2021-10-26 at 17 25 53" src="https://user-images.githubusercontent.com/276316/139307690-415b26c3-25df-469d-a396-00eeaeca2ee3.png"> |


## Testing Instructions




This PR can be tested by following these steps:

- Verify that the border width inputs do not have labels at the bottom when unlocked.
- Verify that the border width inputs do not have a separator gap in gutters between each input.
- Verify that each border width input has a corresponding suffix icon denoting the border side it controls as per [Figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor-Sprint-Feature-Level-Designs_?node-id=10250%3A436799) .
- Verify the border panel matches the design  on [Figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor-Sprint-Feature-Level-Designs_?node-id=10250%3A436799)

 
- Verify the border width still works as expected.

## Reviews

### Does this PR have a security-related impact?
No



### Does this PR change what data or activity we track or use?
No



### Does this PR have a legal-related impact?
No



## Checklist




---



Fixes #9461
